lib/db/db-pool.js

```diff
@@ -4,8 +4,9 @@ const knex = require('knex');
 
 module.exports.createDb = function({
     db,
-    poolMin = 2,
-    poolMax = 20,
+    poolMin = db.pool ? db.pool.min || 0 : 2,
+    poolMax = db.pool ? db.pool.max || 4 : 20,
+    idleTimeoutMillis = db.pool ? db.pool.idleTimeoutMillis || 30000 : 30000,
```

Comment:
This is unfortunate, but the tests started failing with the new OSS (min: 0, max: 4), so this is a compromise since if you go through the normal startup call,  db.pool.min and db.pool.max will be set, but if you come from our tests, `db.pool` will be null, and as such we'll fallback to the value after the `:`, whereas if db.pool is set, but no min|max (which shouldn't happen) we'll use the new defaults of 0 and 4

Reply:
Turned out the tests; thought to be set to min: 1 and max: 1, actually had gotten the default of min: 2 and max: 20 since they used minPool and maxPool instead of poolMin and poolMax on the outermost config object.

In order for the db.stores.destroy() not to fail with aborted, I needed to configure the dbInit code for tests to have a min of 2 and max of 8. It could probably be worth it to see why we need so many connections (I suspect we might be destroying the stores in parallel and as such need a connection per store we're destroying?)